Vendor/CocoaAsyncSocket/GCDAsyncSocket.m

```diff
@@ -1210,14 +1210,16 @@ - (dispatch_queue_t)delegateQueue
 
 - (void)setDelegateQueue:(dispatch_queue_t)newDelegateQueue synchronously:(BOOL)synchronously
 {
+   __block typeof(self) bSelf = self;
```

Comment:
We are not using self in the block so don't need this line anymore.

Reply:
done